### PR TITLE
Cerberus as a Python package

### PR DESCRIPTION
**Pull request by yashashreesuresh**
Cerberus as a Python package to ease installation.
Fixes: #63

### CODE REVIEW

**Comment by yashashreesuresh** [comment]
Please have a look @chaitanyaenr

File: README.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ Cerberus watches the Kubernetes/OpenShift clusters for dead nodes, system compon
...
+#### Usage
+To start cerberus monitoring, execute:
+
+`cerberus_client -c <config_file_location>`
+
+**NOTE**: When config_file_location is not passed, default config file is used.
+
 ### Install the dependencies
 ```
 $ pip3 install -r requirements.txt
```
Comment (paigerube14) [inline README.md:26]: Do we need to still install dependencies if we use the pip instal cerberus-client. I think that we should maybe specify you can use the python and do these new steps you added or clone the repo and you will have to install the requirements. Thoughts?

**Comment by yashashreesuresh** [comment]
> I am getting an error when running. Not sure if this is related to these updates. Will log onto my jump host and see if the error continues
> 
> objc[10556]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
> objc[10556]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.

I am not sure what this error is and why it arose.

File: README.md (hunk 1 above)
Comment (yashashreesuresh) [reply README.md:26]: No, we need not install dependencies, it's automatically installed.

Comment (paigerube14) [reply README.md:26]: Yes, ok! The way it's currently laid out is a little confusing that it still says to install dependencies.

I think adding in a title just saying 2 possibilities of installation.

Comment (chaitanyaenr) [reply README.md:26]: +1 for adding that there are two ways to install and use cerberus.

Comment (yashashreesuresh) [reply README.md:34]: Changed it. Thank you.

Comment (paigerube14) [reply README.md:34]: Looks awesome! Thanks

**Pull request merged**